bosh-stemcell/spec/stemcells/ubuntu_xenial_spec.rb

```diff
@@ -13,7 +13,7 @@
       its(:content) { should match %r{kernel /boot/vmlinuz-\S+-generic ro root=UUID=} }
       its(:content) { should match ' selinux=0' }
       its(:content) { should match ' cgroup_enable=memory swapaccount=1' }
-      its(:content) { should match ' console=ttyS0,115200n8 console=tty0' }
+      its(:content) { should match %r{console=ttyS0,115200n8 console=(hvc|tty)0} }
```

Comment:
its unnecessary to add the xenial related specs in the bionic track

Reply:
The PR has been adapted to remove Xenial specifics.